pretyping/evarconv.ml

```diff
@@ -110,11 +110,13 @@ type flex_kind_of_term =
   | MaybeFlexible of EConstr.t (* reducible but not necessarily reduced *)
   | Flexible of EConstr.existential
 
+let has_arg s = Option.has_some (Stack.strip_n_app 0 s)
```

Comment:
Seems like this would be nicer implemented directly in Stack

Reply:
It essentially was, but the point is to reduce the size of the API. Most of the Stack API must disappear eventually, and this is clearly redundant. I have another series of patches forecast anyways.